mysql/datadog_checks/mysql/mysql.py

```diff
@@ -490,6 +513,30 @@ def _get_server_pid(self, db):
 
         return pid
 
+    def _get_is_aurora(self, db):
+        """
+        Tests if the instance is an AWS Aurora database and caches the result.
+        """
+        if self._is_aurora is not None:
+            return self._is_aurora
+
+        try:
+            with closing(db.cursor()) as cursor:
+                cursor.execute(SQL_SERVER_ID_AWS_AURORA)
+                if len(cursor.fetchall()) > 0:
+                    self._is_aurora = True
+                else:
+                    self._is_aurora = False
+
+        except Exception:
+            self.warning(
```

Comment:
can we return false here instead of implicitly returning None and handling None as false in the check method?

Reply:
Yes good call.